src/menu-button.js

```diff
@@ -62,7 +63,7 @@ class MenuButton extends LitElement {
   render() {
     return html`
       <a
-        href="${this.href}"
+        href="${ifDefined(this.href)}"
```

Comment:
Nice! I've needed this before.

Reply:
Yeah, I noticed when I was hovering over these that they had an href of undefined, and I remembered this from lit-html a while back.